### PR TITLE
adding footer

### PR DESCRIPTION
**Pull request by NagaDurga16**
Footer added

image

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -62,7 +62,8 @@ export default function RootLayout({
             <Content style={contentStyle}>{children}</Content>
           </Layout>
           <Footer style={footerStyle}>
-            <Typography>&copy; Meesho </Typography>
+            {/* <Typography>&copy; Meesho </Typography> */}
```
Comment (NagaDurga16) [inline src/app/layout.tsx]: Remove this

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
```
Comment (NagaDurga16) [inline package.json:12]: Revert this change

File: src/app/componentts/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                  src="https://images.meesho.com/images/pow/appstore-icon-big_400.webp"
+                />
+              </a>
+            </div>
+          </div>
+        </Col>
+        <Col className="gutter-row" span={4}>
+          <div>
+            <a
+              href="https://www.meesho.io/jobs?utm_medium=footer&utm_source=meesho_website&utm_campaign=careerspagepromotion"
```
Comment (NagaDurga16) [inline src/app/componentts/footer.tsx]: Remove meesho links

File: src/app/componentts/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+            </Typography>
+            <Typography style={{ fontSize: 15, color: "gray" }}>
+              Trusted by more than 1 Crore Indians
+            </Typography>
+            <Typography style={{ fontSize: 15, color: "gray" }}>
+              Cash on Delivery | Free Delivery
+            </Typography>
+            <div style={{ display: "Flex" }}>
+              <a
+                href="https://play.google.com/store/apps/details?id=com.meesho.supply&pid=pow_website&c=pow&pow_click_page_type=PDP&pow_click_page_title=kids%20lehenga%20choli&pow_distinct_id=18c8ffd1329ff7-0fe42a5de8979a-17462c6f-1fa400-18c8ffd132a1f43"
```
Comment (NagaDurga16) [inline src/app/componentts/footer.tsx]: Remove all links linked to other websites

**Review by NagaDurga16 — COMMENTED** [review]
please address the review comments.

**Comment by NagaDurga16** [comment]
updated footer
Screenshot from 2023-12-23 12-15-36

File: src/app/componentts/footer.tsx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                  style={{ width: 160 }}
+                  src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQt4QcOwLdFYst6geIybh_PnL38ci6zL8fb1w&usqp=CAU"
+                />
+              </a>
+            </div>
+          </div>
+        </Col>
+        <Col className="gutter-row" span={4}>
+          <div>
+            <a
```
Comment (arajajyothibabu) [inline src/app/componentts/footer.tsx]: Replace all <a> with `<Link>` from `next/link`.

File: src/app/componentts/footer.tsx [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+              href=""
+              target="_blank"
+            >
+              <Typography style={{ color: "gray", fontSize: 15 }}>
```
Comment (arajajyothibabu) [inline src/app/componentts/footer.tsx]: Dedupe the code with a new component , say: 
```
function FooterLink(name: string, href: string)
```

File: src/app/componentts/footer.tsx [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+        </Col>
+        <Col className="gutter-row" span={4}>
+          <div>
+            <Typography style={{ fontSize: 20 }}>Reach out to us</Typography>
+            <div
+              style={{
+                display: "Flex",
+                justifyContent:"space-between"
+              }}
+            >
```
Comment (arajajyothibabu) [inline src/app/componentts/footer.tsx]:
```suggestion
            <Flex justify="space-between">
```

**Comment by NagaDurga16** [comment]
Screenshot from 2023-12-27 11-32-38
Screenshot from 2023-12-27 11-33-46

**Review by Namahsivaya7 — APPROVED** [review]

**Review by tulasikurete — APPROVED** [review]

File: src/app/components/footer.tsx [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
...Wy+BCZx2ywwodrKKdJQHbjRzgmfp52kbw2u+KYxaTgHQpCybqeGSZeAfnNHx8FEn+xOcp/2NIY/g/ooeuLBMjVmXA4L85YQti1ZkUV+CAojIQ1xu8zKf+DYnCnuJ81KGkIOFMD/0WK8Vu68KA+Nki6uDN/+Zx/arFhIEMKKwDMvpElkHWmI0cWJhyKBdT3AuVPvSVcHAcBw3hQuQsuKayMCedrpNpajw0C7/TTUF1DrsY4TuAR7U4Ts0hkejZIurAw/uLOD5tQvuHPkh6n1qJ2QlgeN4wY0IpCnIW2HGHAo63byp9H1+B4mGqw54ZSe4aTj6VLSGjutE3GjpJl1LEqz1Ogc26LK8NvqmIk2uRsnj2WqGFTWX9FOBV8yS6OevkaKdDou9hNj3BPlcX22UcuFiJZA+aSlJueNF/joig6aUxCm0uXzUaqS7JLjanDJFUAYR9GjeI1+J6SkYbQY7G9zC52HyVIsu0iTboJVjNoBRxWw/Qjt7FhvkHQedHUHzcu0ZJukTbUNJRPzmgvYYzsav4L+sE+2qvacP3q5MchGQ9kqlLQ373Qin+z57IxUaDbBR8zz7Y9pp9OhD5bm2HDJVTvERzIc0GWVi80TNbe+R85H13LQaa17nueNC59IE+/JvXddjT2vl/t8QYA3SrQKzLlROKP66+GB52XfPpJHxik8mKox0v9tyEE3rFXy2oqEJ6Ux2L0F3QK7XjQ3tw5T2ne8+e4wGaqm8SJpHAOKvPdtSiSKNHZ9DaQ50u8ac9CNCULvqB1ITNpY7zz+Hmpnel4Kjmhuvbw3DTrohkRbLpOttDIhMsn3WzUl0ri3d6610OQakP97e0IOGhS1XPr0La9xsCn6uzqmmZpd1/S7d+9Ow29E6qaSnhvz1Jig2RLsPOTc1+r1koagc8B0Ey/d333IuZOWVpov27Yt9w/5tZ9bTdPyowTTOnvSWAPTDB7+a2zvA+nTkCB9LFX/9oX4nks9Vf6ei/hGUU1V/kaR+O5WTVX/7pb4llwt1fiWnPg+Yi3V+D6i+OZnHdX65qf4jm111fuOrfg2c1XV/jaz+N54Bd3ve+NkPg9m/dAfp38kct1o8lE9ycpUbwntl75QjP0498AWuovui1lISEhISEhISEhISEhISEhISEhISEhISEhISEhISOjR6//MgZ8r7dDfygAAAABJRU5ErkJggg=="
```
Comment (arajajyothibabu) [inline src/app/components/footer.tsx]: replace this with url

File: src/app/globals.css [hunk 9]
```diff
@@ -105,3 +105,21 @@ a {
...
+      display: none;
+  } */
+  /* .footerstyle{
+    
+    flex-direction:row;
+    flex-wrap: wrap;
+  } */
+}
+
+
```
Comment (arajajyothibabu) [inline src/app/globals.css]: remove this unused CSS

File: src/app/layout.tsx [hunk 10]
```diff
@@ -62,7 +62,7 @@ export default function RootLayout({
             <Content style={contentStyle}>{children}</Content>
           </Layout>
           <Footer style={footerStyle}>
-            <Typography>&copy; Meesho </Typography>
+            <FooterComponent/>
```
Comment (arajajyothibabu) [inline src/app/layout.tsx]:
```suggestion
            <Footer />
```

**Review by arajajyothibabu — COMMENTED** [review]
Small NITs, otherwise LGTM

File: src/app/layout.tsx [hunk 11]
```diff
@@ -61,9 +62,9 @@ export default function RootLayout({
             ></Sider>
             <Content style={contentStyle}>{children}</Content>
           </Layout>
-          <Footer style={footerStyle}>
-            <FooterComponent/>
-          </Footer>
+          <Layout.Footer style={footerStyle}>
+            <Footer/>
+          </Layout.Footer>
```
Comment (arajajyothibabu) [inline src/app/layout.tsx]: Why do we need Layout.Footer here ? This is already used in Footer component

**Review by arajajyothibabu — COMMENTED** [review]
Footer file is deleted somehow, get it back in the branch.

**Review by arajajyothibabu — APPROVED** [review]
🎉
